tests/e2e/workloads/pgsql/test_pgsql_ocp_shutdown.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs import constants
+from tests.sanity_helpers import Sanity
+from ocs_ci.framework.testlib import (
+    E2ETest, workloads, ignore_leftovers
+)
+from ocs_ci.ocs.pgsql import Postgresql
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def pgsql(request):
+
+    pgsql = Postgresql()
+
+    def teardown():
+        pgsql.cleanup()
+    request.addfinalizer(teardown)
+    return pgsql
+
+
+@ignore_leftovers
+@workloads
+@pytest.mark.polarion_id("OCS-820")
+class TestPgSQLNodeShut(E2ETest):
+    """
+    Test running PGSQL and with Ceph pods respin
+    """
+    @pytest.fixture()
+    def pgsql_setup(self, pgsql):
+        """
+        PGSQL test setup
+        """
+        # Deployment of postgres database
+        pgsql.setup_postgresql(replicas=3)
+
+        # Initialize Sanity instance
+        self.sanity_helpers = Sanity()
+
+    @pytest.mark.parametrize(
```

Comment:
I think this has to be removed. Seems to be copy/paste issue from the previous test.